app/Libraries/OsuAuthorize.php

```diff
@@ -895,7 +895,7 @@ public function checkChatPmStart(?User $user, User $target): string
             $this->ensureHasPlayed($user);
         }
 
-        if ($user->isModerator()) {
+        if ($user->isModerator() || $user->isBot()) {
```

Comment:
why is this one missing `isAdmin` compared to the one in `App\Models\Chat\Channel#isVisibleFor` :thinking: 

is it time for a new `isThing` check

Reply:
because it's `OsuAuthorize` 👀